project/Paradox.scala

```diff
@@ -30,11 +30,11 @@ object Paradox {
         "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
         "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
         "extref.ecs.base_url" -> "https://example.lightbend.com/v1/download/%s",
-        "scaladoc.akka.base_url" -> "https://doc.akka.io/api/akka/2.8",
+        "scaladoc.akka.base_url" -> s"https://doc.akka.io/api/akka/${version.value}",
```

Comment:
We shouldn't link to the exact patch version. In other projects we have `AkkaVersionInDocs` for the minor. Can probably be extracted from the `version`.

Reply:
Alright, I'll extract minor version (I started out like that but then thought that we probably delete any API docs in parallel with reference docs so would be fine).